tests/post/steps/test_authentication.py

```diff
@@ -81,25 +81,64 @@ def control_plane_ip(host):
 
 # Given {{{
 
+
 @given(parsers.parse(
     "the control-plane Ingress path '{path}' is available"))
```

Comment:
What this `path` is used for ?

Reply:
True... here is why: https://github.com/scality/metalk8s/pull/2213/files#diff-6065c212c4eed6c66088271ca7d2eddfR142